kornia/contrib/models/efficient_vit/model.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+import torch
+
+from kornia.contrib.models.base import ModelBase
+from kornia.contrib.models.efficient_vit import backbone as vit
+from kornia.core import Tensor
+
+
+@dataclass
+class EfficientViTConfig:
+    """Configuration to construct EfficientViT model.
+
+    Args:
+        checkpoint: URL or local path of model weights.
+    """
+
+    checkpoint: str = "https://huggingface.co/kornia/efficientvit_imagenet_b1_r224/resolve/main/b1-r224.pt"
+
+
+class EfficientViT(ModelBase[EfficientViTConfig]):
+    """EfficientViT backbone model."""
+
+    def __init__(self, backbone: vit.EfficientViTBackbone | vit.EfficientViTLargeBackbone) -> None:
+        super().__init__()
+        self.backbone = backbone
+
+    @staticmethod
+    def from_config(config: EfficientViTConfig) -> EfficientViT:
+        """Build the EfficientViT model from a configuration object.
+
+        Args:
+            config: EfficientViT configuration object.
```

Comment:
add a note here about the checkpoint need to have the pattern into the name, otherwise, it will fail to retrieve the filename and model type